CONTRIBUTING.md

```diff
@@ -1,57 +1,196 @@
 ## Welcome
-We're glad you're thinking about contributing to an 18F open source project! If you're unsure about anything, ask us — or submit the issue or pull request anyway. The worst that can happen is that we’ll politely ask you to change something. 
+We're glad you're thinking about contributing to an 18F open source project!
+If you're unsure about anything, ask us — or submit the issue or pull request
+anyway. The worst that can happen is that we’ll politely ask you to change
+something.
 
-We love all friendly contributions, and we welcome your ideas about how to make the FEC's online presence more user friendly, accessible, and elegant. 
+We love all friendly contributions, and we welcome your ideas about how to
+make the FEC's online presence more user friendly, accessible, and elegant.
 
-To ensure a welcoming environment for our projects, our staff follows the [18F Code of Conduct](https://github.com/18F/code-of-conduct/blob/master/code-of-conduct.md); contributors should do the same. Please also check out the [18F Open Source Policy GitHub repository]( https://github.com/18f/open-source-policy). 
+To ensure a welcoming environment for our projects, our staff follows the
+[18F Code of Conduct](https://github.com/18F/code-of-conduct/blob/master/code-of-conduct.md);
+contributors should do the same. Please also check out the
+[18F Open Source Policy GitHub repository]( https://github.com/18f/open-source-policy).
+
+If you’d prefer, you can also reach us by
+[email](mailto:betafeedback@fec.gov).
 
-If you’d prefer, you can also reach us by [email](mailto:betafeedback@fec.gov).
 
 ## Public domain
-By submitting a pull request, you agree to comply with the policies on our [LICENSE](LICENSE.md) page.
+By submitting a pull request, you agree to comply with the policies on our
+[LICENSE](LICENSE.md) page.
+
 
 ## Issues
-We use GitHub issues to track user issues and team tasks. Whenever possible, we follow this outline:
+We use GitHub issues to track user issues and team tasks. Whenever possible,
+we follow this outline:
  
-1. Goal: a quick blurb explaining the bug or what the issue should accomplish. What is the user need?
+1. Goal: a quick blurb explaining the bug or what the issue should accomplish.
+   What is the user need?
```

Comment:
We made user need optional a while ago. We found that it was kind of awkward for a lot of dev tasks, but it makes sense for a lot of the design tasks.